package.json

```diff
@@ -43,28 +43,28 @@
   },
   "dependencies": {
     "@cerner/terra-aggregate-translations": "^2.0.1",
-    "@cerner/terra-application-docs": "^2.0.0",
+    "@cerner/terra-application-docs": "^3.1.0",
     "@cerner/terra-core-docs": "^1.0.0",
-    "@cerner/terra-dev-site": "^7.0.0",
+    "@cerner/terra-dev-site": "^8.0.0",
     "@cerner/terra-docs": "^1.0.0",
     "@cerner/terra-framework-docs": "^1.0.0",
     "@cerner/terra-graphs-docs": "^1.0.1",
     "@cerner/terra-toolkit-docs": "^2.0.0",
     "classnames": "^2.2.5",
     "final-form": "^4.11.1",
+    "node-sass": "^6.0",
```

Comment:
Is it necessary to pull in node-sass?

Reply:
No, it's unnecessary. It was made mostly for testing - why project isn't running after **npm start**